README.md

```diff
@@ -40,13 +43,52 @@ jobs:
     
     steps:
       - name: Clean up the GitHub package versions that were created for this branch
-        uses: im-open/delete-branch-package-versions@v1.0.2
+        uses: im-open/delete-branch-package-versions@v2.0.0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          organization: 'myOrg'
+          branch-name: ${{ github.head_ref }}
+          package-type: 'npm'
+          package-names: 'my-pkg'
+```
+
+*Delete versions of multiple packages*
+```yml
+on:
+  pull_request:
+    types: [closed]
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Clean up the GitHub package versions that were created for this branch
+        uses: im-open/delete-branch-package-versions@v2.0.0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          organization: 'myOrg'
+          branch-name: ${{ github.head_ref }}
+          package-type: 'npm'
+          package-names: 'my-pkg, another-pkg'
+```
```

Comment:
Might it be useful to include a NuGet example? Maybe switch this one to NuGet?

```suggestion
*Delete versions of multiple packages*
```yml
on:
  pull_request:
    types: [closed]
jobs:
  cleanup:
    runs-on: ubuntu-latest
    
    steps:
      - name: Clean up the GitHub package versions that were created for this branch
        uses: im-open/delete-branch-package-versions@v2.0.0
        with:
          github-token: ${{ secrets.GITHUB_TOKEN }}
          organization: ${{ github.repository_owner }}
          branch-name: ${{ github.head_ref }}
          package-type: 'nuget'
          package-names: 'Some.Package.Name, Another.Package'
```
```